### ISSUE TITLE
noise in adjacent sound clips

### ISSUE THREAD

**Issue by fabrixx** [OpenShot/openshot-qt#1455]
Arch Linux
OpenShot 2.4.1.1

I found this annoying problem in my last job: when I join two audio tracks together (I tried with wav + wav wav + mp3) the resulting audio in preview and exported file is disturbed, has a constant buzz in the background. After trying dozens of output audio formats converting with ffmpeg I finally arrived at the solution: If I separate the two audio tracks slightly, the noise disappears but returns in correspondence of some deadlifts between video tracks. I solved by going to move even the problematic video tracks of a few milliseconds.

**Comment by MarcelWaldvogel** [comment]
I had scratching noise in the nightly at transitions (preview+export); probably not the same as you had. Going back to 2.4.1 (Linux AppImage x86_64) solved the problem for me.

**Comment by fabrixx** [comment]
At the moment it is impossible for me to start the appimage due to this error:

```
Got keys from plugin meta data ("xcb") 
QFactoryLoader::QFactoryLoader() checking directory path "/tmp/.mount_rhGEQg/usr/platforms" ... 
loaded library "/tmp/.mount_rhGEQg/usr/bin/platforms/libqxcb.so" 
QLibraryPrivate::loadPlugin failed on "/tmp/.mount_rhGEQg/usr/bin/platforms/libqxcb.so" : "Cannot load library /tmp/.mount_rhGEQg/usr/bin/platforms/libqxcb.so: (/usr/lib/libfontconfig.so.1: undefined symbol: FT_Done_MM_Var)" 
This application failed to start because it could not find or load the Qt platform plugin "xcb".
Available platform plugins are: eglfs, kms, linuxfb, minimal, minimalegl, offscreen, xcb.
Reinstalling the application may fix this problem.
Annullato (core dump creato)
```

**Comment by peanutbutterandcrackers** [comment]
@fabrixx - Good sir, could you please do a screencast of this issue so that this can be understood even better, and easily? I'll try to get the devs attention to this issue. At least a screenshot or two?

**Comment by fabrixx** [comment]
A screenshot in my opinion would be irrelevant because it is to remove or distance two clips. However, I can link a video that I have exported. The problem can be seen after 3:01 minutes. I was able to limit the problem to the last seconds by spacing and cutting the clips after the title but the problem on the last break despite all my attempts remained with a buzzzz sound:
https://youtu.be/xW6wMaVJWiM?t=176

**Comment by peanutbutterandcrackers** [comment]
I have just notified Mr. Thomas regarding this. Hope this will be resolved soon.

It would have been nice to see  in your screencast how the tracks and the clips are laid up, though.

**Comment by fabrixx** [comment]
I think this my issue:
https://github.com/OpenShot/openshot-qt/issues/1186

**Comment by DylanC** [comment]
@fabrixx Can you try v2.4.2 and let us know if you have any success?

**Comment by DylanC** [comment]
@fabrixx - Oh I can see you already have this solved in #1186. Closing this out then.
